Doppler.HtmlEditorApi/Controllers/CampaignsController.cs

```diff
@@ -133,9 +133,13 @@ public async Task<IActionResult> SaveCampaign(string accountName, int campaignId
             else
             {
                 await _campaignContentRepository.CreateCampaignContent(accountName, contentRow);
+                // We are using HTMLSourceType = 2 (Template) because Editor seems to be tied to the old HTML Editor
+                // See https://github.com/MakingSense/Doppler/blob/48cf637bb1f8b4d81837fff904d8736fe889ff1c/Doppler.Transversal/Classes/CampaignHTMLContentTypeEnum.cs#L12-L17
+                await _campaignContentRepository.UpdateCampaignStatus(2, 2, contentRow.CampaignId, 1);
```

Comment:
What do you think about using a constant in place of the magic number?

```suggestion
                // We are using HTMLSourceType = 2 (Template) because Editor seems to be tied to the old HTML Editor
                // See https://github.com/MakingSense/Doppler/blob/48cf637bb1f8b4d81837fff904d8736fe889ff1c/Doppler.Transversal/Classes/CampaignHTMLContentTypeEnum.cs#L12-L17
                await _campaignContentRepository.UpdateCampaignStatus(2, TemplateHtmlSourceType, contentRow.CampaignId, 1);
```

BTW, what do you think about making honor to the previous format with the parameter names to make it clear?

```suggestion
                // We are using HTMLSourceType = 2 (Template) because Editor seems to be tied to the old HTML Editor
                // See https://github.com/MakingSense/Doppler/blob/48cf637bb1f8b4d81837fff904d8736fe889ff1c/Doppler.Transversal/Classes/CampaignHTMLContentTypeEnum.cs#L12-L17
                await _campaignContentRepository.UpdateCampaignStatus(
                    setCurrentStep: 2, 
                    setHtmlSourceType: TemplateHtmlSourceType, 
                    whenIdCampaignIs: contentRow.CampaignId, 
                    whenCurrentStepIs: 1);
```

Reply:
Looks good 👍